sidecar/src/shm_remote_config.rs

```diff
@@ -37,6 +37,26 @@ use zwohash::ZwoHasher;
 pub struct RemoteConfigWriter(OneWayShmWriter<NamedShmHandle>);
 pub struct RemoteConfigReader(OneWayShmReader<NamedShmHandle, CString>);
 
+/// Function to dump the invariants and target, and the corresponding path.
+/// This is useful for debugging purposes.
+///
+/// # Safety
+/// Pointers should be valid and non-null.
+#[no_mangle]
+pub unsafe extern "C" fn debug_dump_inv_tar(
```

Comment:
Does anything prevent you from putting the ffi functions into actual sidecar-ffi crate?

Reply:
Sounds fine, thanks.